clinicadl/callbacks/factory/early_stopping.py

```diff
@@ -172,5 +173,10 @@ def check_list(value) -> list:
             )
 
     def on_epoch_end(self, config: _TrainingState, **kwargs):
+        all_stop = True
         for metric in self.early_config_list:
             metric.on_epoch_end(config=config, **kwargs)
+            if not metric.stop:
+                all_stop = False
+
+        config.stop = all_stop
```

Comment:
```suggestion
    def on_epoch_end(self, config: _TrainingState, **kwargs):
        return all(metric.on_epoch_end(config=config, **kwargs) for metric in self.early_config_list)
```
And you keep all the ``returns`` in ``OneMetricEarlyStopping``. WDYT?